File: Java/day6.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("enter-x coord: ");
        double x = sc.nextDouble();
        System.out.print("enter-y coord: ");
        double y = sc.nextDouble();
        if (x > 0 && y > 0) {
            System.out.println("(" + x + ", " + y + ") is in Q1");
        } else if (x < 0 && y > 0) {
            System.out.println("(" + x + ", " + y + ") in Q2");
        } else if (x < 0 && y < 0) {
            System.out.println("(" + x + ", " + y + ") in Q3");
        } else if (x > 0 && y < 0) {
            System.out.println("(" + x + ", " + y + ") in Q4");
        } else if (x == 0 && y != 0) {
            System.out.println("(" + x + ", " + y + ") on the y-axis");
        } else if (x != 0 && y == 0) {
            System.out.println("(" + x + ", " + y + ") on the x-axis");
        } else {
            System.out.println("(" + x + ", " + y + ") at origin");
        }
        sc.close();
    }
}

File: Python/day6.py
#Write a program to find the Quadrants in which coordinates lie
x=int(input("Enter X Coordinate: "))
y=int(input("Enter Y Coordinate: "))
#Quadrant I has positive x and y values, Quadrant II has negative x and positive y, Quadrant III has negative x and y, 
#and Quadrant IV has positive x and negative y.
if (x>=0 and y>=0 ):
    print(f"({x},{y}) are in Q1")
elif (x<0 and y<0):
    print(f"({x},{y}) are in Q3")
elif(x<0 and y>0):
    print(f"({x},{y}) are in Q2")
else:
    print(f"({x},{y})are Q4")

# def quad(x, y):
#     if x == 0 and y == 0:
#         return "Origin"
#     elif x == 0:
#         return "Y-axis"
#     elif y == 0:
#         return "X-axis"
    
#     switch = {
#         (True, True): "I Quadrant",    
#         (False, True): "II Quadrant",  
#         (False, False): "III Quadrant",
#         (True, False): "IV Quadrant"   
#     }
#     return switch[(x > 0, y > 0)]
    
# x = int(input("Enter X Coord: "))
# y = int(input("Enter Y Coord: "))
# res = quad(x, y)
# print(f"({x}, {y}) are in the {res}")
